app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/check_claim_summary.scala.html

```diff
@@ -0,0 +1,111 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@import play.twirl.api.Html
+@import play.api.i18n.Messages
+@import play.api.data.Form
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.ViewConfig
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.Claim
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.routes
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterClaimController.CheckClaimAnswer
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryList
+@import cats.data.NonEmptyList
+@import uk.gov.hmrc.govukfrontend.views.Aliases.RadioItem
+@import uk.gov.hmrc.govukfrontend.views.Aliases.Text
+
+@this(
+        layout: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.Layout,
+        claimSummaryHelper: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.utils.ClaimSummaryHelper,
+        submitButton: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.button,
+        pageHeading: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.page_heading,
+        formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF,
+        radios: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.input_radio,
+        summary: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.summary,
+        paragraph: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.paragraph_block,
+        errorSummary: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.error_summary
+)
+
+
+@(claims: NonEmptyList[Claim], form: Form[CheckClaimAnswer])(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
```

Comment:
you seem to have two templates: `check_claim_summary` and `check_claim` ?

Reply:
check_claim was Gabor's, now removed.